config/resources/defaults.yaml

```diff
@@ -15,7 +15,8 @@
 #
 
 Debug: false
-
+Logging:
+  LogLevel: "Error"
```

Comment:
I kind of like just keeping the name `LogLevel` since if the user is changing it withing their Pelican.yaml it won't really be considered the "default level" anymore. However, if you would really like this change I can implement it pretty easily.

Reply:
Maybe we can simplify to `Logging.Level` instead of `Logging.LogLevel`?